txp/Pos/Txp/Toil/Types.hs

```diff
@@ -116,6 +117,9 @@ instance Default MemPool where
 type UtxoModifier = MM.MapModifier TxIn TxOutAux
 type UndoMap = HashMap TxId TxUndo
 
+fromUtxo :: Utxo -> UtxoModifier
```

Comment:
I don't like this name. There are many things which can be constructed from `Utxo`.
`utxoToModifier` maybe? We have `utxoToStakes`, for instance.
Moreover, I think conversion from `Map k v` to `MapModifier k v` should be in `Modifier.hs` rather than here (it's more general).